packages/rocketchat-api/server/v1/subscriptions.js

```diff
@@ -0,0 +1,24 @@
+RocketChat.API.v1.addRoute('subscriptions.get', { authRequired: true }, {
```

Comment:
Would be to hard to have the same response format for calls with and without the updatedAt filter?

Reply:
Nope, not at all.